packages/scene-composer/src/components/three-fiber/Fog.tsx

```diff
@@ -0,0 +1,27 @@
+import * as THREE from 'three';
+import { FC, useContext, useEffect } from 'react';
+import { useThree } from '@react-three/fiber';
+
+import { useStore } from '../../store';
+import { IFogSettings, KnownSceneProperty } from '../../interfaces';
+import { sceneComposerIdContext } from '../../common/sceneComposerIdContext';
+
+const Fog: FC = () => {
```

Comment:
This can be simplified as follows,
```
if (!fogSettings || !fogSettings.colorHex) {
    scene.fog = null;
    return null;
  }

  const fog = new THREE.Fog(fogSettings.colorHex, fogSettings.near, fogSettings.far);

  useEffect(() => {
    scene.fog = fog;
  }, [scene, fog]);
};
```

Reply:
Okay using the ternary